build.sbt

```diff
@@ -634,14 +639,22 @@ val isCE3 = settingKey[Boolean]("Is the current build using CE3?")
 
 lazy val Dependencies = new {
 
+  object ScalaJS {
+    val scalaJsJavaSecureRandom: Def.Initialize[ModuleID] = Def.setting(
+      ("org.scala-js" %%% "scalajs-java-securerandom" % "1.0.0").cross(
+        CrossVersion.for3Use2_13
```

Comment:
I'm not sure if we should add this (I remember seeing that using `for3Use2_13` was not appropriate for libraries)

Reply:
I recommend using `cats.effect.std.UUIDGen` and avoiding this dep altogether. See:
- https://github.com/typelevel/cats-effect/pull/2945

Reply:
I will attempt to rewrite, but we depend on cats-effect transitively so I'll have to update the build a bit. In the mean time, FYI @Baccata :

![Screen Shot 2022-04-13 at 11 17 15](https://user-images.githubusercontent.com/5230460/163213855-c667d137-7081-42d8-b09a-8e16ba1e9a67.png)

Reply:
my latest changes should do it!